src/util.h

```diff
@@ -13,4 +13,8 @@
 **/
 #define ARRAY_SIZE(arr)     ( sizeof(arr) / sizeof((arr)[0]) )
 
+
+static inline Int AlwaysYes( Obj obj) { return 1; }
+static inline Int AlwaysNo( Obj obj) { return 0; }
+
```

Comment:
OK, that extra space definitely should go (not sure how I missed that, yikes). Of course clang-format overall suggest this diff:
```diff
diff --git a/src/util.h b/src/util.h
index b17a7cc8b..fdcf6c139 100644
--- a/src/util.h
+++ b/src/util.h
@@ -11,10 +11,16 @@
 **
 ** Compute the number of elements of a given C array.
 **/
-#define ARRAY_SIZE(arr)     ( sizeof(arr) / sizeof((arr)[0]) )
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof((arr)[0]))


-static inline Int AlwaysYes( Obj obj) { return 1; }
-static inline Int AlwaysNo( Obj obj) { return 0; }
+static inline Int AlwaysYes(Obj obj)
+{
+    return 1;
+}
+static inline Int AlwaysNo(Obj obj)
+{
+    return 0;
+}

-#endif // GAP_UTIL_H
+#endif    // GAP_UTIL_H
```
I am fine with doing either one; and in addition I could also move AlwaysYes / AlwaysNo into a new `src/util.c` file... Thoughts?